ingesters/GooglePubSubIngester/main.go

```diff
@@ -34,123 +28,42 @@ const (
 	defaultConfigLoc  = `/opt/gravwell/etc/pubsub_ingest.conf`
 	defaultConfigDLoc = `/opt/gravwell/etc/pubsub_ingest.conf.d`
 	appName           = `pubsub`
+	ingesterName      = "GooglePubSub"
 )
 
 var (
-	configLoc      = flag.String("config-file", defaultConfigLoc, "Location of configuration file")
-	confdLoc       = flag.String("config-overlays", defaultConfigDLoc, "Location for configuration overlay files")
-	verbose        = flag.Bool("v", false, "Display verbose status updates to stdout")
-	ver            = flag.Bool("version", false, "Print the version information and exit")
-	stderrOverride = flag.String("stderr", "", "Redirect stderr to a shared memory file")
-	lg             *log.Logger
+	lg *log.Logger
+	v  bool
```

Comment:
Having a global named "v" is probably going to bite us, as "v" is also the idiomatic loop variable when using `range`. We'll certainly shadow this variable in places.

I suggest we change all of these to "verbose" (renaming the flag variable), or just use the flag variable that's already there as we were doing before.

Reply:
agreed, fixed, PTAL